routes/movies.routes.js

```diff
@@ -0,0 +1,92 @@
+const router = require("express").Router();
+const Celebrity = require("../models/Celebrity.model");
+const Movie = require("../models/Movie.model");
+
+router.get("/movies/create", (req, res, next) => {
+  res.render("movies/new-movie");
+});
+
+router.post("/movies/create", async (req, res, next) => {
+  try {
+    const { title, genre, plot } = req.body;
+    await Movie.create({ title, genre, plot });
+    res.redirect("/movies");
+  } catch (error) {
+    console.log(error);
+    next(error);
+    res.redirect("/new-movie");
+  }
+});
+
+router.get("/movies", async (req, res, next) => {
+  try {
+    let movies = await Movie.find();
```

Comment:
And here you would need use populate!

`let movies = await Movie.find().populate('cast')`